src/github.com/robertkrimen/otto/.gitignore

```diff
@@ -1,5 +0,0 @@
-/.test
```

Comment:
Ah, my bad.  I saw it in proxiedsites.go rather than proxiedsites_test.go and assumed it was part of the production code.